internal/bminventory/inventory.go

```diff
@@ -4789,7 +4792,7 @@ func (b *bareMetalInventory) UpdateHostInstallProgress(ctx context.Context, para
 		}
 
 		log.Info(fmt.Sprintf("Host %s in cluster %s: %s", host.ID, host.ClusterID, event))
-		eventgen.SendHostInstallProgressUpdatedEvent(ctx, b.eventsHandler, *host.ID, *host.ClusterID, hostutil.GetHostnameForMsg(&host.Host), event)
+		eventgen.SendHostInstallProgressUpdatedEvent(ctx, b.eventsHandler, host.ClusterID, *host.ID, *host.ClusterID, hostutil.GetHostnameForMsg(&host.Host), event)
```

Comment:
looks we can replace the clusterID with infraenvid here as well

Reply:
done